Neos.Flow/Classes/Configuration/RouteConfigurationProcessor.php

```diff
@@ -197,6 +197,12 @@ protected function buildSubRouteConfigurations(array $routesConfiguration, array
                         $mergedSubRouteConfiguration['defaults'][$key] = $this->replacePlaceholders($defaultValue, $variables);
                     }
                 }
+                foreach ($mergedSubRouteConfiguration['routeParts'] ?? [] as $routePartKey => $routePartValue) {
+                    $mergedSubRouteConfiguration['routeParts'][$routePartKey] = $this->replacePlaceholders($routePartValue, $variables);
```

Comment:
Don't we need to check if `$routePartValue` is actually a string? In a lot (most?) cases it definitely is an array (see next line)

Reply:
Doh, good point.. I'll fix

Reply:
I just removed the line, it didn't do anything really.. Good catch!

Reply:
I thought it was meant for hardcoded route parts... 
```yml
-
  name: 'Some route'
  uriPattern: 'some/{part}'
  routeParts:
    'part': '<fixedPartVariable>'
```
which doesn't make much sense if I think about it again :D

Reply:
No, it was targeted at the `handler` itself:
```yaml
-
  name: 'Some route'
  uriPattern: 'some/{part}'
  routeParts:
    'part':
      handler: '<variable>'
      options: []
```
but was misspelled and that usecase seems too random anyways